src/common/utils/delay.ts

```diff
@@ -1,6 +1,8 @@
-// Create async delay for provided timeout in milliseconds
-
-export async function delay(timeoutMs = 1000) {
-  if (!timeoutMs) return;
-  await new Promise(resolve => setTimeout(resolve, timeoutMs));
+/**
```

Comment:
Why this change is needed? :) 
Also I would prefer to keep `timeoutMs` as it is more clear/descriptive without consulting the docs..

Reply:
I had this function as part of my code base before it go merged to master. I can revert the `timeoutMs` in a followup PR if you want.

Though side note, what editor are you using? I use vscode and it always shows the full docs (including the tsdocs) when I hover over a function.

Reply:
I use `WebStorm` and this is what I see:

![image](https://user-images.githubusercontent.com/6377066/107392157-10268a80-6b02-11eb-8161-f8c9128d39d2.png)

Reply:
Hovering also works, but I don't use it usually (if I need to read docs I'd better navigate to the source):

![image](https://user-images.githubusercontent.com/6377066/107392657-9511a400-6b02-11eb-9fd5-905f23c9db44.png)